pkg/resources/fluentd/statefulset.go

```diff
@@ -48,6 +48,23 @@ func (r *Reconciler) statefulset() (runtime.Object, k8sutil.DesiredState) {
 }
 
 func (r *Reconciler) statefulsetSpec() *appsv1.StatefulSetSpec {
+	initContainers := make([]corev1.Container, 0)
+
+	if !r.Logging.Spec.FluentdSpec.DisablePvc && r.Logging.Spec.FluentdSpec.VolumeMountChmod {
```

Comment:
Do we need the PVC part for this condition?

Reply:
not really, I've removed it